integrations/aws/utils/resources.py

```diff
@@ -128,32 +128,42 @@ async def resync_custom_kind(
         describe_method_params = {}
     while True:
         async with session.client(service_name) as client:
-            params: dict[str, Any] = describe_method_params
-            if next_token:
-                params[marker_param] = next_token
-            response = await getattr(client, describe_method)(**params)
-            next_token = response.get(marker_param)
-            results = response.get(list_param, [])
-            logger.info(
-                f"Fetched batch of {len(results)} from {kind} in region {region}"
-            )
-            if results:
-                yield [
-                    {
-                        CustomProperties.KIND: kind,
-                        CustomProperties.ACCOUNT_ID: account_id,
-                        CustomProperties.REGION: region,
-                        **fix_unserializable_date_properties(resource),
-                    }
-                    for resource in results
-                ]
-
-        if not next_token:
-            break
+            try:
+                params: dict[str, Any] = describe_method_params
+                if next_token:
+                    params[marker_param] = next_token
+                response = await getattr(client, describe_method)(**params)
+                next_token = response.get(marker_param)
+                results = response.get(list_param, [])
+                logger.info(
+                    f"Fetched batch of {len(results)} from {kind} in region {region}"
+                )
+                if results:
+                    yield [
+                        {
+                            CustomProperties.KIND: kind,
+                            CustomProperties.ACCOUNT_ID: account_id,
+                            CustomProperties.REGION: region,
+                            **fix_unserializable_date_properties(resource),
+                        }
+                        for resource in results
+                    ]
```

Comment:
lets log also that we received empty results, same as my previous pr for regular kinds

Reply:
There's already a log to indicate the amount of resources received, Do you think we should also add log to indicate empty results specifically?